euler-maven-1/src/main/java/com/ceritoxi/Math.java

```diff
@@ -0,0 +1,23 @@
+package com.ceritoxi;
+
+public class Math {
+	
+	public static boolean isDividableByXorY(int dividend, int numberX, int numberY) {
+		if (numberX == 0 && numberY == 0) return false ;
```

Comment:
Always use code block separators (curly) even if there is just one line in it such as:
```
if (numberX == 0 && numberY == 0) {
 return false ;
}
```